.vscode/settings.json

```diff
@@ -3,5 +3,25 @@
   "editor.formatOnSave": true,
   "editor.defaultFormatter": "esbenp.prettier-vscode",
   "typescript.enablePromptUseWorkspaceTsdk": true,
-  "typescript.tsdk": "node_modules/typescript/lib"
+  "typescript.tsdk": "node_modules/typescript/lib",
```

Comment:
Hmmm why?

Reply:
Agh this should be in the .gitignore sorry

Reply:
Well I do like the base settings. I think that's helpful so it updates when you save. I just didn't understand the updates related to this PR. It's fine to remove for now